src/cloud-cli/userdb.ts

```diff
@@ -156,18 +156,18 @@ export function migrate(): number {
 
   if (create_db) {
     logger.info(`Creating database ${userdbname}`);
-    const cmd = `createdb -h ${configFile.database.hostname} -p ${configFile.database.port} ${userdbname} -U ${configFile.database.username} -w ${configFile.database.password} -e`;
```

Comment:
Even better, let's just create a client and run a `CREATE DATABASE IF NOT EXISTS`